### PR TITLE
Update status check result messaging

### PR DESCRIPTION
**Pull request by krischarbonneau**
### Description
This PR updates the text for each status check result to be in line with the latest content as per the above ADO task.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to `/status`
4. Fill in the form using data (either from DB or mock data) for each status code and confirm the content matches what is in the ADO task

### CODE REVIEW

File: components/CheckStatusResponses/CheckStatusShippingCanadaPost.tsx [hunk 1]
```diff
@@ -13,25 +13,28 @@ export const CheckStatusShippingCanadaPost: FC<
 
   return (
     <>
-      <p data-testid="shipped-canada-post">
-        {t('shipped-canada-post.printed-and-mailing')}
-      </p>
+      <h2 data-testid="shipped-canada-post" className="h2 text-blue-normal">
+        {t('status-check-passport-printed')}
+      </h2>
+      <span>{t('shipped-canada-post.mailing')}</span>
```
Comment (sebastien-comeau) [inline components/CheckStatusResponses/CheckStatusShippingCanadaPost.tsx:19]: Should be wrapped with a `p` tag.

```tsx
<p>
        {t('shipped-canada-post.mailing')}
        {trackingNumber && (
          <>
            {t('status-check-tracking.number')} <b>{trackingNumber}</b>.<br />
            <ExternalLink
              href={t('status-check-tracking.link.canada-post', {
                trackingNumber,
              })}
            >
              {t('status-check-tracking.link.text')}
            </ExternalLink>
          </>
        )}
      </p>
```

`\n` shouldn't be used for the link text

File: components/CheckStatusResponses/CheckStatusShippingFedex.tsx [hunk 2]
```diff
@@ -12,20 +12,21 @@ export const CheckStatusShippingFedex: FC<CheckStatusShippingFedexProps> = ({
   const { t } = useTranslation(['status', 'common'])
   return (
     <>
-      <p data-testid="shipped-fedex">
-        {t('shipped-fedex.printed-and-mailing')}
-      </p>
+      <h2 data-testid="shipped-fedex" className="h2 text-blue-normal">
+        {t('status-check-passport-printed')}
+      </h2>
+      <span>{t('shipped-fedex.mailing')}</span>
```
Comment (sebastien-comeau) [inline components/CheckStatusResponses/CheckStatusShippingFedex.tsx:18]: Should be wrapped with a `p` tag.

```tsx
<p>
        {t('shipped-fedex.mailing')}
        {trackingNumber && (
          <>
            {t('status-check-tracking.number')} <b>{trackingNumber}</b>.<br />
            <ExternalLink
              href={t('status-check-tracking.link.fedex', { trackingNumber })}
            >
              {t('status-check-tracking.link.text')}
            </ExternalLink>
          </>
        )}
      </p>
```

`\n` shouldn't be used for the link text

**Review by sebastien-comeau — COMMENTED** [review]
Trivial HTML changes.

File: components/CheckStatusResponses/CheckStatusShippingCanadaPost.tsx (hunk 1 above)
Comment (krischarbonneau) [reply components/CheckStatusResponses/CheckStatusShippingCanadaPost.tsx:19]: Fair enough, I was originally using `<br />` to force the new line but figured the `whitespace-pre-line` was a cleaner alternative

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**